### ISSUE TITLE
Hydronic Network Sizing

### ISSUE THREAD

**Issue by MichaMans** [RWTH-EBC/uesgraphs#4]
We had some utilities functions like the sizing of a hydronic network for heating (and maybe cooling?). Is it planned to reintegrate this or what do you think about such helper functions for the graph framework?
@marcusfuchs

**Comment by marcusfuchs** [comment]
@MichaMans Good question. I am a bit divided on this issue. On the one hand such functions can be helpful. On the other hand, this is a special case when we consider the general graph framework for all possible network types (including e.g. electricity). To me, it seems hard to add this in a general way. Therefore, maybe it should rather be outside the core of the framework? But I also see the advantages of including it for convenience. In any case, unfortunately I currently do not  have the bandwidth to work on this topic. Therefore I have no plans in this direction, but I am open to helping with the integration if someone wants to add this.

**Comment by MichaMans** [comment]
@marcusfuchs Yeah! Good to know. I find it hard too, but in my opinion it would be a great addition. Of course we should implement this for heating and cooling networks to have a wide spectrum of usage. In general i think the more that tool can provide the better it is for the use of a urban energy systems graph framework. But maybe we'll see it in a different angle in the future. For now I'll try to make a suggestion.
